plugin/aliasing.go

```diff
@@ -0,0 +1,97 @@
+package gcpauth
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/iam/v1"
+)
+
+type aliasData struct {
```

Comment:
This doesn't appear to be used anywhere?

Reply:
Removed. Leftover from a previous design that I scrapped.